scripts/setup-local-dev.sh

```diff
@@ -32,7 +31,8 @@ rm -rf node_modules;
 yarn;
 yarn run build;
 yarn link;
-$SHOULD_SETUP_MARKETS && yarn run deploy:environment;
+
+docker cp $CONTAINER_NAME:/augur.js/src/contracts/addresses.json src/contracts/addresses.json;
```

Comment:
I guess nice to have (correct contracts) but not needed if using :latest, just need latest augur.js linked in to augur-node and augur ui.

Reply:
Same reason as above. I want to support working on an old branch.